CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 -
 
 ### Changed
--
+- **cf-core:** [MINOR] Remove inline-block style from inline code.
```

Comment:
Actually, this should probably be a patch.

Reply:
Fixed!